prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -481,29 +481,25 @@ def prompt_processor() -> Any:
             if assertion_failed or answer.lower() == "[]" or answer.lower() == "na":
                 structured_output[output[PSKeys.NAME]] = None
             else:
-                prompt = (
-                    f"Convert the following text:\n{answer} into valid JSON format."
-                )
+                prompt = f"Convert the following text into valid JSON string: \
+                    \n{answer}\n\n The JSON string should be able to be parsed \
+                    into a Python dictionary. \
+                    Output just the JSON string. No explanation is required. \
+                    If you cannot extract the JSON string, output {{}}"
```

Comment:
@tahierhussain why do we ask the LLM to outptut `{{}}` ?
Rather than relying on the LLM for this, we can try to call
```
try:
    json.loads(json_candidate_from_llm)
except JSONDecodeError as e:
    answer = {{}} # explicitly set the default required
```

Reply:
@chandrasekharan-zipstack I'm assigning empty object `{}` to the result in the catch block. This should handle the case. Please check.

Reply:
@tahierhussain if we default it to `{}` in the catch block, why do we ask the LLM to default it to `{{}}` ? I'm still not clear on that. Anyway I think `{{}}` would result in a `JSONDecodeError` so its fine - I'd still like to know the reason, and if its not really required we should keep things simple and let it be `{}` in the prompt as well

Reply:
@chandrasekharan-zipstack We're asking the prompt to return empty object `{}` as default. In f-string, we cannot add empty object `{}` directly, that's why I've added it like this `{{}}`. You can read more about it [here](https://stackoverflow.com/questions/5466451/how-do-i-escape-curly-brace-characters-in-a-string-while-using-format-or).